tutorials/access-mac-mini-with-reemo/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+
+## Reeemo - Overview
+
+[Reemo](https://reemo.io/) developed a remote desktop solution to access your computers from anywhere on any device without any configuration. 
+Once the Reemo service is installed on the remote device you can acccess it from any device running a Chrome browser. 
+
+<Message type="requirement">
+
+You have:
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+- You have [created a Mac mini M1](/compute/apple-silicon/how-to/create-mac-mini-m1/)
+</Message>
+
+## Creating your Reemo account.
+
+<Message type="tip">
+
+A Reemo Cloud account account is required to access your Mac mini M1 via the Reemo service. Reemo Cloud is **free** for personal use.
+
+</Message>
+
+1. Go to the [Reemo portal](https://portal.reemo.io/) and click **Get started** to create your Reemo account.
+
+  <Lightbox="scaleway-reemo-portal.png" alt="" />
+
+2. Enter your email address, a password and its confirmation in the form. Then tick the checkbox to agree Reemo's terms of service and click **Contine** to create your account. You will receive an email with a confirmation code.
+
+3. Open your email account and check your security code. Then enter the code in the form and click **Validate** to confirm the account creation. 
+
+  <Lightbox="scaleway-reemo-validate.png" alt="" />
+
+  <Message type="tip">
+
+  If you don not receive the validation code, check also in your spam folder. If you still can not find the code, click **Send again** to receive it again.
+
+  </Message>
+
+4. Click **My private key** to display your private key which you need during the installation of the Reemo client on your Mac mini M1. 
+
+  <Message type="important">
+
+  Keep this code secret and don't share it.
+
+  </Message>
+
+## Installing the Reemo client
+
+1. Log into your Mac mini M1. You can [connect](/compute/apple-silicon/how-to/connect-to-mac-mini-m1/) to your machine either using the VNC client or via SSH. 
+
+2. If you are logged into the Mac mini M1 via VNC, open a terminal. If you are connected via SSH you can skip this step.
+
+3. Run the following command to install the Reemo client on your machine:
+  ```sh 
+  curl -skL -o /tmp/install_reemo.x 'https://download.reemo.io/macos/setup.x' && sudo bash /tmp/install_reemo.x
+  ```
+  <Message type="note">
+
+  You must be logged into the administrator account of your Mac mini M1 to run the command above.
+
+  </Message>
+
+4. Enter the **private key** of your Reemo account when prompted. Press Enter to validate.
+
+The Reemo client is installed now on your Mac mini M1 and the machine appears on the Reemo dashboard.
+
+## Connecting to your Mac mini M1 using Reemo
+
+1. Open the [Reemo portal](https://portal.reemo.io/) in a [Chrome](https://www.google.com/chrome/) browser on a device of your chocice. 
+
+  <Message type="tip">
+
+  Reemo works on any device capable to run the Chrome browser. No matter if it is your PC, tablet, phone or any device.
```

Comment:
```suggestion
  Reemo works on any device capable of running the Chrome browser. This could be your PC, tablet, phone or any other device running Chrome.
```